### ISSUE TITLE
[bitnami/mysql] master/slave podAnnotations dont used in templates/

### ISSUE THREAD

**Issue by evgkrsk** [bitnami/charts#1854]
**Which chart**:

bitnami/mysql-6.6.0

**Description**

There is **master.podAnnotations** and **slave.podAnnotations** in **values.yaml/values-production.yaml*, but there is NO references to **.Values.master.podAnnotations** or **.Values.slave.podAnnotations** in **templates/** directory of helm chart. As result, master/slave.podAnnotations in release values has no effect.

**Steps to reproduce the issue:**

1. Install helm chart with master.podAnnonations dict in values.yaml
2. Check master pod annotations

**Describe the results you received:**

No annotations on pods

**Describe the results you expected:**

Annotations from values.yaml present on pods

**Additional information you deem important (e.g. issue happens only occasionally):**

I think that there is simple typo in template/{master,slave}-statefulset.yaml : must be **.Values.master.podAnnotations** instead of **.Values.master.annotations** and **.Values.slave.podAnnotations** instead on **.Values.slave.annotations**

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.0.2", GitCommit:"19e47ee3283ae98139d98460de796c1be1e3975f", GitTreeState:"clean", GoVersion:"go1.13.5"}
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-13T11:23:11Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.3", GitCommit:"b3cbbae08ec52a7fc73d334838e18d17e8512749", GitTreeState:"clean", BuildDate:"2019-11-13T11:13:49Z", GoVersion:"go1.12.12", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by alemorcuq** [comment]
Thanks for reporting this, @evgkrsk! We will fix it.

**Comment by alemorcuq** [comment]
Fixed in version `6.7.6`.
